python/llm/src/bigdl/llm/transformers/convert.py

```diff
@@ -173,6 +173,15 @@ def optimize(model):
                         module.SelfAttention,
                         chatglm_attention_forward
                         )
+    elif "gptj" in model.config.model_type:
+        modeling_module_name = model.__class__.__module__
+        module = importlib.import_module(modeling_module_name)
+        if "GPTJForCausalLM" in model.config.architectures:
```

Comment:
do we need this? it seems true for all gpt-j models (e.g., https://huggingface.co/EleutherAI/gpt-j-6b)

Reply:
removed.